types/react-virtual-keyboard/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for react-virtual-keyboard 1.0
+// Project: https://www.npmjs.com/package/react-virtual-keyboard
+// Definitions by: Bogdan Surai <https://github.com/bsurai>
```

Comment:
@sheetalkamat Hi. Thanks. I'm not sure what do you mean. There is correct reference to react-virtual-keyboard npm package. What is wrong? Do I need to add READ.me ?